packages/ui/src/components/content/NewsArticleCard.vue

```diff
@@ -1,23 +1,31 @@
 <script setup lang="ts">
-import dayjs from "dayjs";
+import dayjs from 'dayjs'
 
-interface Article {
-  path: string;
-  thumbnail: string;
-  title: string;
-  summary: string;
-  date: string;
+export interface Article {
+  path: string
+  thumbnail: string
+  title: string
+  summary: string
+  date: string
 }
 
 defineProps<{
-  article: Article;
-}>();
+  article: Article
+}>()
+
+const emit = defineEmits<{
+  view: [article: Article]
+}>()
+
+const handleClick = (article: Article) => {
+  emit('view', article)
+}
 </script>
 
 <template>
-  <nuxt-link
-    :to="`${article.path}/`"
-    class="active:scale-[0.99]! group flex flex-col transition-all ease-in-out hover:brightness-125"
+  <div
+    class="active:scale-[0.99]! group flex flex-col transition-all ease-in-out hover:brightness-125 cursor-pointer"
+    @click="handleClick(article)"
```

Comment:
this is not great, links should be real links. You should be good to use <router-link> from vue-router in this package

Reply:
Are we sure? During the servers refactor stuff last week it didn't work for the web frontend.

Reply:
yeah, we use it all over the place

Reply:
Fair, will change it now.